### ISSUE TITLE
[Android/Windows Mobile] Dragging the map triggers the infowindow

### ISSUE THREAD

**Issue by Xatpy** [CartoDB/cartodb.js#346]
It's only happening with **Android** (Chrome) and **Window Mobile** (Internet Explorer 11), so it works on iOS (Chrome/Safari).

This issue comes from [here](https://github.com/CartoDB/cartodb.js/issues/223) : 
(Tested on [this map](https://find.cartodb.com/viz/19dd0966-65b2-11e4-8337-0e853d047bba/public_map))

![pan_infowindow_mobile](https://cloud.githubusercontent.com/assets/3958416/6215140/327e4344-b5fe-11e4-9951-6c8397a3f26f.gif)

**Comment by Xatpy** [comment]
- I was debugging with iPhone 4S / iOS7(Safari Dev Tools) and Android/OnePlus One (Chrome Dev Tools).
- **The issue is that with Android/Chrome, touchmove is been triggered a lot of times (while you're clicking on the map). However, iOS/Safari only triggered once this event. I've recorded a video with the issue: [Link](https://dl.dropboxusercontent.com/u/22131567/WP_20150217_003.mp4)**
- Then, I checked that the 'problem' was with Leaflet. I created an [example](http://bl.ocks.org/Xatpy/raw/646ba7d8d1da3a177a0f/) (only mobile). The key is: it only shows the infowindow if the map doesn't move. And we're not doing that (we always launch the infowindow).
- So, I realised that we were interacting with Leaflet maps wrongly. We launch the infowindow directly, and we don't know if the map has been moved or not. I mean: we are triggering onUp event directly, instead of waiting for finger goes out. 
- I think that the issue is [here](https://github.com/CartoDB/cartodb.js/blob/develop/vendor/wax.cartodb.js#L3066). I've replaced it with `bean.add(parent(), 'touchend', onUp);`and it seems to work, but I've to test it tomorrow (and talking with Jamón).
